doc/source/multiprocessing.rst

```diff
@@ -0,0 +1,38 @@
+Multiprocessing API on Ray
```

Comment:
this needs to be in linked in `index.rst`

Reply:
or actually specifically, `using-ray.rst`

Reply:
Will do. Haven't finished writing docs yet I'll ask for your review when I do

Reply:
I don't think this should go in `using-ray.rst` - that seems focused on ray core APIs but this is more of a library.